modules/auxiliary/scanner/http/totaljs_traversal.rb

```diff
@@ -0,0 +1,136 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
```

Comment:
```suggestion
# This module requires Metasploit: https://metasploit.com/download
```

Reply:
Sorry, I started from a template taken from the official documentation. I provide to fix